prow/config/jobs_test.go

```diff
@@ -129,37 +132,122 @@ func TestCommentBodyMatches(t *testing.T) {
 	}{
 		{
 			"org/repo",
-			"this is a random comment",
+			"/this-is-a-random-comment gce",
```

Comment:
There is no need to change `TestCommentBodyMatches`, as this is purely a synthetic test with no real data. 

AKA if we don't update the `c := &Config{}` definition we also don't need to update the `var testcases = []struct{}` definition, saving us a bunch of churn